### ISSUE TITLE
Half disabled split button arrow does not have enough contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#9251]
The enabled arrow is using disabled text color on themePrimary, making it nearly invisible.
![image](https://user-images.githubusercontent.com/20363037/58518993-7587eb80-8166-11e9-8199-e2718714844b.png)
